src/main/webapp/app/course/competencies/competency-management/competency-import-course.component.html

```diff
@@ -0,0 +1,59 @@
+<form>
```

Comment:
Please check if the generalized `import.component.html` template can be used here.

Reply:
Sadly the generalized importComponent cannot be used here (afaik) since I need to use some additional checkboxes. If you know a way how this could be handled please let me know :)

Reply:
It would however be a good Idea to maybe make the `import.component.html` use another component for the table - This way we could re-use that for import dialogs that are slightly different. Would be a good idea for a follow-up PR